ram/wram.asm

```diff
@@ -668,16 +668,24 @@ wDebugDarkTileColor::  ds 2
 wDebugBlackTileColor:: ds 2
 
 NEXTU
-wc608:: ds 16
-wc618:: ds 37
+wc608:: ds 5
+wc60d:: ds 2
```

Comment:
This address is the start of a mobile `party_struct`. I believe its a temporary location, so it will probably need to be unionized. 

I'd recommend adding this struct as `wMobileMon`, such as:

```
wMobileMon:: ds party_struct wMobileMon
```

Some of routines that use this party struct are the following:

- Function17d0f3
- Function11b6b4
- Function11b7e5

Reply:
I had a party_struct there but decided against it until c608 got a union. here's also one at wc633 and s5_a82c
until its better documented the MON_ macros seemed cleaner

Reply:
c608 is already part of a union. You can do this:

```diff
+NEXTU
+	ds 5
+wMobileMon:: ds party_struct wMobileMon
+
 NEXTU
 wc608:: ds 16
 wc618:: ds 37
 ...
 wc6a8:: ds 40
 ENDU
```